ajax/libs/qoopido.demand/package.json

```diff
@@ -34,5 +38,8 @@
     "type": "git",
     "url": "https://github.com/dlueth/qoopido.demand"
   },
-  "license": "(MIT OR GPL-3.0+)"
+  "licenses": [
+    "MIT",
+    "GPL-3.0"
```

Comment:
I think it should be `"deprecated_GPL-3.0+"` in SPDX format.

https://github.com/cdnjs/cdnjs/blob/12d79a4bf555f80bd5b8c1b694ccf127e69aea0c/tools/license-list.json#L119